Source/Fuse.Text/Implementation/ICU.cpp.uxl

```diff
@@ -1,31 +1,60 @@
-	<CopyFile Condition="MSVC" Name="../icu/x64/icu/bin64/icuuc55.dll" TargetName="bin/x64/Debug/icuuc55.dll" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icudt55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icuin55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icuio55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icule55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/iculx55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icutu55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icuuc55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icudt55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icuin55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icuio55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icule55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/iculx55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icutu55.dll':Path)" />
+	<Require Condition="WIN32 && X86" SharedLibrary="@('../icu/x86/icu/bin/icuuc55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icudt55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icuin55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icuio55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icule55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/iculx55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icutu55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icuuc55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icudt55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icuin55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icuio55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icule55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/iculx55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icutu55.dll':Path)" />
+	<Require Condition="WIN32 && X64" SharedLibrary="@('../icu/x64/icu/bin64/icuuc55.dll':Path)" />
+	<!-- Legacy -->
+	<CopyFile Condition="TARGET_MSVC" Name="../icu/x86/icu/bin/icudt55.dll" TargetName="bin/Win32/Release/icudt55.dll" />
```

Comment:
What's the point of keeping these?

Reply:
We still need these because the MSVC target isn't actually removed yet (https://github.com/fusetools/uno/pull/1382). We can remove these later when the new stuff has landed.

Reply:
Why is this different from V8Simple, then?

Reply:
It's not different - V8Simple already had the CopyFile TARGET_MSVC elements, so wasn't much to do there...

Reply:
To clarify:
- `MSVC` is still defined by the new native target.
- `TARGET_MSVC` is only defined by the legacy MSVC target.
- We don't want to copy these files when using the new target, so we now use `TARGET_MSVC` to avoid breaking the old target until it's been completely removed.

Reply:
I still don't understand why V8Simple doesn't need a lecacy CopyFile-line. You deleted that in this commit, and added a `<Require ... SharedLibrary="..." />`-line. For ICU, you duplicated this stuff instead.

Reply:
Here are V8Simples CopyFile lines
```
	<CopyFile Condition="TARGET_MSVC" Name="lib/Windows/x86/V8Simple.dll" TargetName="bin/Win32/Debug/V8Simple.dll" />
	<CopyFile Condition="TARGET_MSVC" Name="lib/Windows/x86/V8Simple.dll" TargetName="bin/Win32/Release/V8Simple.dll" />
	<CopyFile Condition="TARGET_MSVC" Name="lib/Windows/x64/V8Simple.dll" TargetName="bin/x64/Debug/V8Simple.dll" />
	<CopyFile Condition="TARGET_MSVC" Name="lib/Windows/x64/V8Simple.dll" TargetName="bin/x64/Release/V8Simple.dll" />
```
https://github.com/mortend/fuselibs-public/blob/dc23e81de07bdfd22f3efe40ebd123bf4abaabc8/Source/Fuse.Scripting.JavaScript/V8/V8Simple.cpp.uxl

They were already there so I didn't have to change anything. For ICU I had to change `MSVC` -> `TARGET_MSVC`.

Reply:
Oh, right. I get it now. Sorry for the confusion :P